plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/ConnectionEditor.java

```diff
@@ -123,7 +127,9 @@ public Composite createControls(Composite parent, ConnectionWizardPageModel page
 
 		//remember token
 		this.rememberTokenObservable = new WritableValue(Boolean.FALSE, Boolean.class);
+		this.rememberPasswordObservable = new WritableValue(Boolean.FALSE, Boolean.class);
 		this.rememberTokenCheckbox = new Button(parent, SWT.CHECK); //parent is reset further down
+		this.rememberPasswordCheckbox = new Button(parent, SWT.CHECK); //parent is reset further down
```

Comment:
widgets in a model? I must have missed this, but it looks odd to me that we have a button in a model.